lib/call.js

```diff
@@ -1578,6 +1578,10 @@ class call {
     If we are not ringing - send ringing to the other end.
   */
   ring() {
+
+    if ( this.established && "uas" === this.type ) {
+      this.channels.audio.play( this.mor )
```

Comment:
Similar comment to below, we should allow for "early" state. Also, should check we have a channel.

I think it is time we have a

```js
play( soup ) {
  if( ! this.channels.audio ) return false
  if( !this.established && !this.early ) return false

  this.channels.audio.play ( soup )
  return true
}
```